globals/globals_test.go

```diff
@@ -3606,8 +3846,6 @@ func TestLoadGlobalsErrors(t *testing.T) {
 					`,
 				},
 			},
-			// FIXME(katcipis): would be better to have ErrGlobalRedefined
```

Comment:
it doesn't make sense to return ErrGlobalRedefined because multiple attributes with the same name is a syntax error.